policyengine_us/variables/gov/states/va/tax/income/deductions/itemized/va_child_dependent_care_deduction.py

```diff
@@ -0,0 +1,21 @@
+from policyengine_us.model_api import *
+
+
+class va_child_dependent_care_deduction(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "VA child/dependent exemption"
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.VA
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.states.va.tax.income.deductions.itemized
+        dependents_count = tax_unit("tax_unit_dependents", period)
+        amount = p.child_dependent_deduction
+
+        return where(
```

Comment:
Please also add a test to ensure it works correctly with two dependents (it wouldn't have previously)
```suggestion
        return amount * where(dependents_count > 1, 2, 1)
```